pkg/scaffold/helm.go

```diff
@@ -234,15 +214,50 @@ func (s *Scaffold) buildChartValues(w *wkspace.Workspace) error {
 	values["plrl"] = map[string]interface{}{
 		"license": w.Installation.LicenseKey,
 	}
+	return values, nil
+}
 
-	io, err := yaml.Marshal(values)
+func fromGoTemplate(vals map[string]interface{}, globals map[string]interface{}, values map[string]map[string]interface{}, w *wkspace.Workspace, chartInst *api.ChartInstallation) error {
```

Comment:
should we perhaps put these template implementations into a subpackage w/in scaffold?